config/share_notify.yml

```diff
@@ -0,0 +1,3 @@
+---
+host: <%= ENV.fetch('APPS_DHH_STAGING_SHARE_NOTIFY_HOST') {'https://staging.osf.io'}%>
```

Comment:
@VivianChu @hweng does this need to be broken up by environment since there are difference URLs/tokens in staging vs prod? Similar to config/solr.yml, for example.